drivers/clock_control/Kconfig.stm32

```
@@ -62,25 +62,54 @@ config CLOCK_STM32_MCO1_SRC_EXT_HSE
 
 config CLOCK_STM32_MCO1_SRC_LSE
 	bool "LSE"
-	depends on SOC_SERIES_STM32F4X
+	depends on SOC_SERIES_STM32F4X || \
+		SOC_SERIES_STM32F7X || \
+		SOC_SERIES_STM32L4X
```

Nit: Indentation.
```suggestion
		   SOC_SERIES_STM32F7X || \
		   SOC_SERIES_STM32L4X
```
Apply everywhere it's applicable

---

fixed.